src/PageContent/PendingAccounts/PendingAccountListItem.js

```diff
@@ -0,0 +1,162 @@
+import React, { Component } from 'react';
+import './PendingAccounts.css';
+import { AccountType } from '../../Enums';
+import VerifyAccount from './VerifyAccount';
+import { accountsRef, donatingAgenciesRef } from '../../FirebaseConfig';
+import ConfirmActivate from './ConfirmActivate';
+import ConfirmReject from './ConfirmReject';
+
+class PendingAccountsListItem extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            dialog: false,
+            acceptPopUp: false,
+            rejectPopUp: false
+        };
+        this.openDialog = this.openDialog.bind(this);
+        this.closeDialog = this.closeDialog.bind(this);
+        this.acceptPopUp = this.acceptPopUp.bind(this);
+        this.back = this.back.bind(this);
+        this.confirmAccept = this.confirmAccept.bind(this);
+        this.confirmReject = this.confirmReject.bind(this);
+        this.rejectPopUp = this.rejectPopUp.bind(this);
+    }
+    
+    openDialog() {
+        this.setState({
+            dialog: true
+        });
+    }
+
+    closeDialog() {
+        this.setState({
+            dialog: false,
+            acceptPopUp: false,
+            rejectPopUp: false
+        });
+    }
+    acceptPopUp() {
+        this.setState({
+            acceptPopUp: true
+        });
+    }
+    rejectPopUp() {
+        this.setState({
+            rejectPopUp: true
+        });
+    }
+
+    confirmAccept(uid, daContactId) {
+        const updates = { isVerified: true, isActivated: true };
+        if (daContactId) { // accountType == donating agency 
+            donatingAgenciesRef.child(uid).update(updates);
+            accountsRef.child(daContactId).update(updates);
+        } else {
+            accountsRef.child(uid).update(updates);
+        }
+        this.setState({
+            acceptPopUp: false,
+            dialog: false,
+        });
+    }
+
+    confirmReject(uid, daContactId) {
```

Comment:
remove from umbrella as well

Reply:
It seems like we have a mix of schemas here. Which one should we use?

<img width="890" alt="screen shot 2018-06-23 at 3 42 56 pm" src="https://user-images.githubusercontent.com/22493572/41814260-4defc4d2-76fc-11e8-95eb-8d3daefdef31.png">

Reply:
Resolved.